pegasus/migrations/100_change_email_in_contacts.rb

```diff
@@ -0,0 +1,15 @@
+# This migration drops the uniqueness constraint on email for the contacts
+# table. As of July 2016, this constraint seems to exist on development
+# environments only.
+
+# WARNING: This migration, though reversible, does not attempt to readd the
+# uniqueness constraint in the down direction.
+
+Sequel.migration do
+  up do
+    alter_table(:contacts) do
+      drop_constraint(:email, :type => :unique)
+    end if DB.indexes(:contacts)[:email] &&
```

Comment:
nit ([ruby-style](https://github.com/bbatsov/ruby-style-guide#no-multiline-if-modifiers)) - `if` conditional should go at the beginning of a multi-line block.